config/translations/lxqt-config-powermanagement_de.ts

```diff
@@ -36,7 +36,7 @@
     <message>
         <location filename="../batterywatchersettings.ui" line="111"/>
         <source>Use icons from theme:</source>
-        <translation>Symbole vom Thema nutzen:</translation>
+        <translation>Symbole des Themas nutzen:</translation>
```

Comment:
bevor wir da was dran machen, sollten wir uns final auf die Übersetzung von 'theme' einigen und das LXQt-weit durchziehen.

Reply:
Ich finde "Thema" passend, da es sich nicht nur um ein spezifisches "Design" handelt, sondern um die Zusammenführung verschiedener Elemente welche jeweils ihr eigenes "Aussehen/Erscheinungsbild/Design" besitzen und im Zusammenspiel eine "Thematik" ergeben. Deshalb finde ich "Thema" sehr passend. Im Englischen ist aus denselben Gründen nicht von "design" sondern "theme" die Rede, was hier ebenfalls das Zusammentreffen unterschiedlicher Elemente beschreibt.

Reply:
gekauft :)

Reply:
Ich weiß nicht, wie es bei Dir dargestellt wird, aber ich habe auf Weblate ein paar Anmerkungen zu Source-Strings gemacht, im Abgleich mit der Laufenden Instanz der jeweiligen Anwendung. Im Idealfall könnte man noch alle bereits erfolgten Übersetzungen von fehlerhaften Source Strings als "needs editing" deklarieren.

Reply:
sollte man - ikke muss mich jetzt erst mal kurz nach Arch begeben und LXQt aktuell neu bauen - script kommt dann rüber. Ich hab gestern erst mal lxqt-archiver eingebunden und die machine-translation getestet, ging erstaunlich gut, aber da müssen wir den Übersetzern noch was an die Hand zu geben - Thema "Best Practices"

Reply:
A few source strings need editing. I left a few comments below of some I encountered on the way. If it could be possible to mark already translated strings as needing an edit, deriving from faulty sources, existing translations could in turn be corrected more easily.

Reply:
An die Hand geben: meinst Du source comments machen?

Reply:
will write an upstream bug about :D - one can set some notifications in user preferences - notification for new translations/comments/suggestions - but comments should get a column in the overview pages.

Reply:
If there are any comments they will be displayed in their chosen scope:
![](http://i.imgur.com/qFWr0se.png)

Reply:
Thank your for the Info.

Reply:
I left a few suggestions in the comments regarding source language of "qterminal":
https://weblate.lxqt.org/translate/lxqt/qterminal/de/?type=comments&offset=0